habitat-lab/habitat/core/utils.py

```diff
@@ -102,22 +115,11 @@ def __call__(cls, *args, **kwargs):
         return cls._instances[cls]
 
 
-def center_crop(obs, new_shape):
-    top_left = (
-        (obs.shape[0] // 2) - (new_shape[0] // 2),
-        (obs.shape[1] // 2) - (new_shape[1] // 2),
-    )
-    bottom_right = (
-        (obs.shape[0] // 2) + (new_shape[0] // 2),
-        (obs.shape[1] // 2) + (new_shape[1] // 2),
-    )
-    obs = obs[top_left[0] : bottom_right[0], top_left[1] : bottom_right[1], :]
-
-    return obs
-
-
 class DatasetJSONEncoder(json.JSONEncoder):
+    """Extension of base JSONEncoder to handle common Dataset types: numpy array, numpy quaternion, Omegaconf, and dataclass."""
+
     def default(self, obj):
+        # TODO: NEED DESCRIPTION
```

Comment:
Need to add a description and a description for "obj".  Looks like the "default" function is a type convertor.